angular-workspace/lighthouserc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
```

Comment:
@Razvan1928 I pushed some changes to the branch and added comments in the top-level PR describing the changes. Let me know if yo have any questions about the changes otherwise I think it's ready to merge.